utilities/blob_db/blob_db_impl.cc

```diff
@@ -948,6 +948,37 @@ Status BlobDBImpl::Write(const WriteOptions& opts, WriteBatch* updates) {
   return Status::OK();
 }
 
+Status BlobDBImpl::GetLiveFiles(std::vector<std::string>& ret,
+                                uint64_t* manifest_file_size,
+                                bool flush_memtable) {
+  ReadLock rl(&mutex_);
```

Comment:
Add comment on why holding mutex at the beginning?

Reply:
to avoid updates to BaseDB during the for loop

Reply:
I mean, add an inline comment to explain?